src/interface/cypress/e2e/accounts.spec.cy.ts

```diff
@@ -0,0 +1,43 @@
+import { randString, urls, baseURL, testPass, testUser } from '../support/util';
+
+const test_email1 = 'localtest' + randString(6) + '@local.example';
+const test_pass1 = 'ABCdef123#$%';
+const test_firstname = 'Larry';
+const test_lastname = 'Larrington';
+
+describe('Create an account with unmatching password', () => {
+  it('Attempts to create an account with bad password', () => {
+    cy.visit(baseURL + urls.SIGNUP);
+    cy.contains('Create your account');
+    cy.get('[formControlName="firstName"]').type(test_firstname);
+    cy.get('[formControlName="lastName"]').type(test_lastname);
+    cy.get('[formControlName="email"]').type(test_email1);
+    cy.get('[formControlName="password1"]').type(test_pass1);
+    cy.get('[formControlName="password2"]').type('Nope!#%');
+    cy.contains('Given passwords must match.');
+    cy.get('button[type="submit"]').should('be.disabled');
+  });
+
+  it('Attempts to create an account', () => {
```

Comment:
This actually creates a test account. It seems important to test, but without having a way to automatically delete these (say, via a script run under `make` or something similar), I've got mixed feelings about leaving this in.